content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -140,20 +171,26 @@ Next, add the following line to your `wrangler.toml` file to point Wrangler to u
 ```diff
   name = "prisma-mongodb-cloudflare"
   type = "webpack"
-+ webpack_config = "webpack.config.js"
++  webpack_config = "webpack.config.js"
```

Comment:
Hm, I added that whitespace because it was breaking the left-alignment of the entries in the Gatsby output. I've now removed all white space so things are aligned.

Thanks for commenting on this, @eemmiillyy.